package.json

```diff
@@ -102,7 +102,7 @@
     "electrum-client": "git+https://github.com/BlueWallet/rn-electrum-client.git",
     "events": "1.1.1",
     "frisbee": "2.0.9",
-    "i18next": "^19.4.5",
+    "i18next": "^19.7.0",
```

Comment:
delete types for i18next https://www.npmjs.com/package/@types/i18next 
<img width="1222" alt="Zrzut ekranu 2020-08-31 o 15 15 26" src="https://user-images.githubusercontent.com/67273035/91723948-d5a4e300-eb9c-11ea-9035-6ac2c7f2550c.png">